deploy.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# This is a simple change.
```

Comment:
Err? Probably shouldn't be here - have you rolled in some test changes from when you're git pushes weren't working?